pkg/app/mutate_app.go

```diff
@@ -255,6 +259,33 @@ func (m *Mutator) mutateLabels(ctx context.Context, app v1alpha1.App, appVersion
 	return result, nil
 }
 
+func findKubeConfigNamespace(ctx context.Context, k8sClient kubernetes.Interface, appNamespace, kubeConfigName string) (string, error) {
+	_, err := k8sClient.CoreV1().Secrets(appNamespace).Get(ctx, kubeConfigName, metav1.GetOptions{})
+	if apierrors.IsNotFound(err) {
```

Comment:
Nitpick: is not better to reverse the condition? If it was found return immediately. If not, then search for it. That way you will avoid the nested `if` and the sudden `return` of a value that was set at the beginning of the func.

Reply:
Yes that is better 👍 thanks